docker/dashboard/Dockerfile

```diff
@@ -1,14 +1,25 @@
-FROM node:20
+FROM node:20 AS base
```

Comment:
Ah, I see this is here. Could we delete the other `Dockerfile` then and just use this one? The rest of the repo only has dockerfiles here.

Reply:
you are right @coltmcnealy-lh I forgot to delete the other file.